src/Security/Authentication/Authenticator.php

```diff
@@ -94,4 +93,14 @@ protected function hasRequiredFields(array $array): bool
 
         return true;
     }
+
+    protected function getRequest(Credentials $credentials): Request
+    {
+        return new Request(
```

Comment:
nah I thought it's easy to implement, I'm not sure if it's so big deal we should add an issue